### ISSUE TITLE
sbt-coursier in project/project/plugins.sbt suppresses global plugins

### ISSUE THREAD

**Issue by fthomas** [fthomas/scala-steward#383]
It seems that having
```sbt
addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M13-1")
```
in `project/project/plugins.sbt` suppresses global plugins.

Here is what I'm observing. I have sbt-updates installed as global plugin:
```
~$ cat .sbt/1.0/plugins/sbt-updates.sbt 
addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.4.0")
``` 
Now if I'm starting sbt in my circe/circe clone (which uses sbt-coursier in `project/project/plugins.sbt`), I can't run sbt-updates' `dependencyUpdates`:
```
~$ sbt dependencyUpdates
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
[info] Loading settings for project global-plugins from metals.sbt,idea.sbt,sbt-updates.sbt ...
[info] Loading global plugins from /home/frank/.sbt/1.0/plugins
[info] Loading settings for project circe-build-build from plugins.sbt ...
[info] Loading project definition from /home/frank/data/code/circe/project/project
[info] Updating ProjectRef(uri("file:/home/frank/data/code/circe/project/project/"), "circe-build-build")...
[info] Done updating.
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[info] Loading settings for project circe-build from plugins.sbt ...
[info] Loading project definition from /home/frank/data/code/circe/project
[info] Loading settings for project circe from build.sbt,version.sbt ...
[info] Resolving key references (24141 settings) ...
[info] Set current project to circe (in build file:/home/frank/data/code/circe/)
[error] Not a valid command: dependencyUpdates
[error] Not a valid project ID: dependencyUpdates
[error] Expected ':'
[error] Not a valid key: dependencyUpdates (similar: dependencyClasspath, dependencyOverrides, dependencyPositions)
[error] dependencyUpdates
[error]                  ^
```

If I comment out the `addSbtPlugin` command in `project/project/plugins.sbt`, sbt-updates `dependencyUpdates` works as expected:
```
~$ sbt dependencyUpdates
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
[info] Loading settings for project global-plugins from metals.sbt,idea.sbt,sbt-updates.sbt ...
[info] Loading global plugins from /home/frank/.sbt/1.0/plugins
[info] Loading settings for project circe-build-build from plugins.sbt ...
[info] Loading project definition from /home/frank/data/code/circe/project/project
[info] Updating ProjectRef(uri("file:/home/frank/data/code/circe/project/project/"), "circe-build-build")...
[info] Done updating.
[info] Loading settings for project circe-build from plugins.sbt ...
[info] Loading project definition from /home/frank/data/code/circe/project
[info] Compiling 1 Scala source to /home/frank/data/code/circe/project/target/scala-2.12/sbt-1.0/classes ...
[info] Done compiling.
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.protobuf.UnsafeUtil (file:/home/frank/.sbt/boot/scala-2.12.7/org.scala-sbt/sbt/1.2.8/protobuf-java-3.3.1.jar) to field java.nio.Buffer.address
WARNING: Please consider reporting this to the maintainers of com.google.protobuf.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[info] Loading settings for project circe from build.sbt,version.sbt ...
[info] Resolving key references (24421 settings) ...
[info] Set current project to circe (in build file:/home/frank/data/code/circe/)
[info] Found 1 dependency update for circe-numbers-testing
[info]   org.scalacheck:scalacheck : 1.13.5 -> 1.14.0
[info] No dependency updates found for circe-core
...
```

Surprinsingly, replacing sbt-coursier in `project/project/plugins.sbt` with another plugin fixes this issue. So it seems specific to sbt-coursier.

**Comment by fthomas** [comment]
Downgrading sbt-coursier to 1.0.3 also fixes this problem. @alexarchambault is this a known problem?

**Comment by alexarchambault** [comment]
@fthomas Yes, it's a known regression in sbt-coursier `1.1.0-M12` (affecting versions up to `1.1.0-M13-1`). It's been fixed by https://github.com/coursier/sbt-coursier/pull/47, included in `1.1.0-M13-2` and subsequent versions.

**Comment by fthomas** [comment]
Thanks for the info @alexarchambault! I've opened a PR in circe to update sbt-coursier.
